lib/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Probably shouldn’t need this file. Should be able to modify the top level eslint config’s overrides to do what you need here (basically adding `lib/*/index.js` to the “node files” entry)...